mappings/net/minecraft/entity/mob/EntityGhast.mapping

```diff
@@ -4,16 +4,21 @@ CLASS zl net/minecraft/entity/mob/EntityGhast
 		METHOD e tick ()V
 	CLASS zl$b
 		METHOD c tick ()V
-	CLASS zl$c
+	CLASS zl$c AiShootFireballs
```

Comment:
Is AiTask a valid naming scheme? The mappings for that seem to be all over the place.